src/main/java/uk/gov/ons/ctp/response/collection/exercise/state/CollectionExerciseStateTransitionManagerFactory.java

```diff
@@ -94,6 +94,17 @@ public CollectionExerciseStateTransitionManagerFactory() {
     transitionForValidated.put(CollectionExerciseEvent.PUBLISH, CollectionExerciseState.READY_FOR_LIVE);
     transitions.put(CollectionExerciseState.VALIDATED, transitionForValidated);
 
+    // FAILEDVALIDATION
+    Map<CollectionExerciseEvent, CollectionExerciseState> transitionForFailedvalidation = new HashMap<>();
+    transitionForValidated.put(CollectionExerciseEvent.RESET, CollectionExerciseState.READY_FOR_LIVE);
```

Comment:
Should this line be:
```java
transitionForFailedvalidation.put(CollectionExerciseEvent.RESET, CollectionExerciseState.READY_FOR_REVIEW);
```
As we want to reset an invalidated collex back to a state where a user can change the samples, instruments, etc.